app/src/main/java/com/calendar/activity/CreateEventActivity.java

```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
             annualCheckBox.setVisibility(View.VISIBLE);
             reminderCheckBox.setVisibility(View.VISIBLE);
             freeCheckBox.setVisibility(View.VISIBLE);
+            showReminderTimeCheck();
```

Comment:
why not use `showReminderClockCheck()`?

Reply:
I didn't want to pass view to the function